examples/chip-tool/commands/pairing/PairingCommand.h

```diff
@@ -184,4 +185,10 @@ class PairingCommand : public CHIPCommand,
     char * mOnboardingPayload;
     uint64_t mDiscoveryFilterCode;
     char * mDiscoveryFilterInstanceName;
+
+    // For unpair
+    chip::Platform::UniquePtr<chip::Controller::CurrentFabricRemover> mCurrentFabricRemover;
+    chip::Callback::Callback<chip::Controller::OnCurrentFabricRemove> mCurrentFabricRemoveCallback;
+
+    static void OnCurrentFabricRemove(void * context, NodeId remoteDeviceId, CHIP_ERROR status);
```

Comment:
```suggestion
    static void OnCurrentFabricRemove(void * context, NodeId nodeId, CHIP_ERROR status);
```

Reply:
I will change to suggestion variable name.